src/cbmc/cbmc_parse_options.cpp

```diff
@@ -110,6 +108,16 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
     exit(CPROVER_EXIT_USAGE_ERROR);
   }
 
+  if(cmdline.isset("paths"))
+    options.set_option("paths", true);
+
+  if(cmdline.isset("goto-priority"))
```

Comment:
Mentioned this below, but make clear that even with `--paths` this can be manually overridden.